packages/p2p/src/components/modal-manager/modals/preferred-countries-modal/preferred-countries-modal.scss

```diff
@@ -0,0 +1,25 @@
+.preferred-countries-modal {
+    &__search-field {
+        margin-bottom: 0;
+
+        input {
+            margin-left: 2.5rem;
+        }
+    }
+
+    &__checkbox {
+        padding: 1.6rem 1rem;
+    }
+}
+
+.dc-modal__container_preferred-countries-modal {
+    .dc-modal-body {
+        padding: 0;
+    }
+
+    .dc-modal-footer {
+        > button {
+            flex: 1;
+        }
+    }
+}
```

Comment:
if we pass className to Modal.Body shouldn't that work instead?

Reply:
is there a way to pass className to Modal.Body?

Reply:
yes you can, there are other Modals in P2P v1 where we pass className directly like this

<img width="653" alt="Screenshot 2024-03-12 at 3 03 38 PM" src="https://github.com/binary-com/deriv-app/assets/103412909/3e64e9cd-3af0-4227-ba93-fb4018d9fbdd">